packages/components/organisms/f-registration/src/components/_tests/Registration.test.js

```diff
@@ -126,9 +142,13 @@ describe('Registration', () => {
         describe('with a faulty registration service', () => {
             it('should populate generic error message and emit failure event when service responds with an error', async () => {
                 // Arrange
-                RegistrationServiceApi.createAccount.mockImplementation(async () => { throw new Error('Conflict'); });
+                RegistrationServiceApi.createAccount.mockImplementation(async () => {
+                    throw new Error('Conflict');
+                });
```

Comment:
Super nit: can you please add an empty line after this?

Reply:
Ill put a refactor ticket in - these are just automatic formatting changes from resolving the previous formatting comment

Reply:
If you do create a refactor ticket for this – we should update the lining/autofix rules to handle this, rather than having to manually check for this kind of thing